src/github.org

```diff
@@ -0,0 +1,104 @@
+#+TITLE: GitHub Training 2017
+#+AUTHOR: Sahit Chintalapudi and Josh Ting 
+#+EMAIL: schintalapudi@gatech.edu
+#+REVEAL_THEME: black
```

Comment:
Please remove these attributes unless you explicily want to override project settings for some reason.